pkg/util/config/config.go

```diff
@@ -67,20 +67,23 @@ type Config struct {
 	Resources
 	Auth
 	Scalyr
-	WatchedNamespace         string            `name:"watched_namespace"` // special values: "*" means 'watch all namespaces', the empty string "" means 'watch a namespace where operator is deployed to'
-	EtcdHost                 string            `name:"etcd_host" default:"etcd-client.default.svc.cluster.local:2379"`
-	DockerImage              string            `name:"docker_image" default:"registry.opensource.zalan.do/acid/spiloprivate-9.6:1.2-p4"`
-	ServiceAccountName       string            `name:"service_account_name" default:"operator"`
-	DbHostedZone             string            `name:"db_hosted_zone" default:"db.example.com"`
-	EtcdScope                string            `name:"etcd_scope" default:"service"`
-	WALES3Bucket             string            `name:"wal_s3_bucket"`
-	KubeIAMRole              string            `name:"kube_iam_role"`
-	DebugLogging             bool              `name:"debug_logging" default:"true"`
-	EnableDBAccess           bool              `name:"enable_database_access" default:"true"`
-	EnableTeamsAPI           bool              `name:"enable_teams_api" default:"true"`
-	EnableTeamSuperuser      bool              `name:"enable_team_superuser" default:"false"`
-	TeamAdminRole            string            `name:"team_admin_role" default:"admin"`
-	EnableLoadBalancer       bool              `name:"enable_load_balancer" default:"true"`
+	WatchedNamespace          string `name:"watched_namespace"` // special values: "*" means 'watch all namespaces', the empty string "" means 'watch a namespace where operator is deployed to'
+	EtcdHost                  string `name:"etcd_host" default:"etcd-client.default.svc.cluster.local:2379"`
+	DockerImage               string `name:"docker_image" default:"registry.opensource.zalan.do/acid/spiloprivate-9.6:1.2-p4"`
```

Comment:
Is it ok to have this version hardcoded here?

Reply:
Outside the scope of this PR, but afaik yes. IMO the reason is that  we want our defaults be applicable across wide variety of installations since users won't change most of them most of the time. From that viewpoint, it makes sense to hardcode a version of Spilo that is certainly stable.